docs/fileset/README.md

```diff
@@ -38,7 +38,40 @@ Each request consists of two parts:
 
 2. **file record** - It contains the actual file.
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
+<template v-slot:py>
+
+``` py
+from jexia_sdk.http import HTTPClient
+
+JEXIA_PROJECT_ID = 'project_id'
+USER_EMAIL = 'user@jexia.com'
+USER_PASSWORD = 'secret-password'
+
+if __name__ == '__main__':
+  client = HTTPClient()
+  client.auth_consumption(
+      project=JEXIA_PROJECT_ID,
+      method='ums',
+      email=USER_EMAIL,
+      password=USER_PASSWORD
+  )
+  data = {
+     "email":"user@email"
+  }
+  files = open('./file.txt','rb')
```

Comment:
Did you test it locally? I think it will not work, file should be read with context manager, like:
```python
...
with open('./file.txt','rb') as file:
    res = client.request(
        method='POST',
        data=user,
        files=file,
        url='/fs/<fileset-name>'
    )
    print(res)
```

Reply:
changed.